### PR TITLE
Add images and video considerations

### PR DESCRIPTION
**Pull request by arzola**
Please review this and add your tricks and recommendations here

### CODE REVIEW

**Review by pabloturati — APPROVED** [review]
Everything looks good;  I am not very knowledgeable about video implementations and therefore not sure how I could further contribute to improving this information but I have certainly found it extremely useful and it seems very clear.

**Comment by imoutaharik** [comment]
I agree! It looks pretty complete. Maybe just to add something... This article explains a bit about html5 videos, formats and browser support. 
https://www.hongkiat.com/blog/html5-videos-things-you-need-to-know/

**Review by imoutaharik — APPROVED** [review]
I agree! It looks pretty complete. Maybe just to add something... This article explains a bit about html5 videos, formats and browser support.
https://www.hongkiat.com/blog/html5-videos-things-you-need-to-know/

File: languages/html.md [hunk 1]
```diff
@@ -1,18 +1,176 @@
...
+  
+
 # HTML Best Practices
 
+  
+
 ## Semantics and Readability
-Be as semantic as possible when structuring the markup of a page. Semantic markup means that tags are chosen based on the type of content and their hierarchy. These are documents and as such should read like a document when the presentation layer is stripped away. Semantic markup also makes it easier to implment accessibility standards when required. 
+
+Be as semantic as possible when structuring the markup of a page. Semantic markup means that tags are chosen based on the type of content and their hierarchy. These are documents and as such should read like a document when the presentation layer is stripped away. Semantic markup also makes it easier to implment accessibility standards when required.
```
Comment (dulcehc) [inline languages/html.md]: There's a typo for "implment", Semantic markup also makes it easier to **implment** accessibility standards when required.

**Review by dulcehc — COMMENTED** [review]
Looks good to me, just a minor typo for "implment".

I found some cool articles to dig deeper about optimization:
1. https://developers.google.com/web/fundamentals/performance/optimizing-content-efficiency/image-optimization
2. https://developers.google.com/web/fundamentals/performance/optimizing-content-efficiency/replace-animated-gifs-with-video/

I think also is good to test the speed of the website in pages like https://developers.google.com/speed/pagespeed/insights/, and check that we don't have problems with images.

File: languages/html.md [hunk 2]
```diff
@@ -1,18 +1,176 @@
...
+  
+
+### General considerations
+
+#### Export final video at high quality
+
+When exporting the final video from your video editing application, be sure to export it with high quality settings. Conversion and optimization for use on the web will be done later (details below).
+
+
+### Using video as a background
```
Comment (adamf321) [inline languages/html.md]: @arzola can you add sample code for a background video - html / css / js?

**Pull request closed without merging**